v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerIT.java

```diff
@@ -109,8 +113,41 @@ public void setUp() throws IOException {
             .setPrivateConnectivity("datastream-private-connect-us-central1")
             .build();
 
-    gcsPrefix = getGcsPath(testName + "/cdc/").replace("gs://" + artifactBucketName, "");
-    dlqGcsPrefix = getGcsPath(testName + "/dlq/").replace("gs://" + artifactBucketName, "");
+    gcsResourceManager = setUpGcsResourceManager();
+    gcsPrefix =
+        getGcsPath(testName + "/cdc/", gcsResourceManager)
+            .replace("gs://" + gcsResourceManager.getBucket(), "");
+    dlqGcsPrefix =
+        getGcsPath(testName + "/dlq/", gcsResourceManager)
+            .replace("gs://" + gcsResourceManager.getBucket(), "");
+  }
+
+  public GcsResourceManager setUpGcsResourceManager() {
+    GcsResourceManager spannerTestsGcsClient;
+    if (TestProperties.project().equals("cloud-teleport-testing")) {
+      List<String> bucketList =
```

Comment:
Would it be possible to make this list as a static list (possibly in a class of it's own or as a part of a testConstants class if we already have one)?

Reply:
done